en/simulation/hitl.md

```diff
@@ -59,12 +59,12 @@ In summary, HITL runs PX4 on the actual hardware using standard firmware, but SI
 
       ![Select Airframe](../../assets/gcs/qgc_hil_config.png)
 1. Setup UDP
-   1. Under the *General* tab of the settings menu, uncheck all *AutoConnect* boxes except for **UDP**. 
+   1. Under the *General* tab of the settings menu, uncheck all *AutoConnect* boxes except for **UDP**. This will disconnect the vehicle from QGC.
```

Comment:
Really? Even if correct, is it useful to note here?

Reply:
It is true because QGC is connected to PX4 via serial port. Serial port's equivalent in QGC is "Pixhawk", which when we uncheck, px4 gets disconnected from QGC.

When I was following these steps, my QGC disconnected and I got confused that something has gone wrong, until a px4 team member clarified that it is normal. So I think, it will be helpful to let the user know that QGC disconnecting at this point is normal and part of the process.

Reply:
Also, the HITL Simulation Environment Diagram above is wrong according to @bkueng. In HITL, there is no serial connection between QGC and PX4 since we uncheck "Pixhawk" in this step. QGC is connected to the Simulator via UDP port and then the Simulator is connected to px4 via serial port. It is something like this. 
@bkueng can you please confirm?

Reply:
No, this is not correct. QGC does not disconnect when unchecking the box. @barzanisar I'm not sure what you refer to, but we experienced another oddity: QGC showed some mavlink error message when we were connected to the Pixracer WiFi and USB at the same time. I don't remember exactly, but I think it was a problem between mavlink 1 and 2.

@barzanisar yes, the updated HITL PR is here: https://github.com/PX4/Devguide/pull/443.

Reply:
@barzanisar Can you remove that line. Other than this, and @RicardoM17 comment (https://github.com/PX4/Devguide/pull/462#discussion_r169814109) this is good to go.